ports/esp32/machine_sdcard.c

```diff
@@ -193,11 +198,15 @@ STATIC mp_obj_t machine_sdcard_make_new(const mp_obj_type_t *type, size_t n_args
     self->flags = 0;
     // Note that these defaults are macros that expand to structure
     // constants so we can't directly assign them to fields.
+
+    int freq_khz = arg_vals[ARG_freq_khz].u_int;
```

Comment:
For coding style, this can go down inside the "else" block below where it's used.